packages/mock-repo/test/eyes.spec.ts

```diff
@@ -0,0 +1,44 @@
+const {makeVisualGridClient, initConfig} = require('@applitools/visual-grid-client');
```

Comment:
Let's not test the grid client in separation, it's a third-party package that should have its own tests, and it's not our job to test third-party code.

If we want to test our code then we need to replace this library with a mock. Our tests should not submit anything to Applitools.